### DISCUSSION TITLE
ColorOS 12 Battery Optimization

### DISCUSSION THREAD

**Discussion by Miquest** [NightscoutFoundation/xDrip#2362] [General]
Hey, l

can you may add this to the Android Configuration?
It's a solution to prevent Apps from killing in the background from "Don't Kill my App".
![Screenshot_2022-10-03-13-46-45-61_40deb401b9ffe8e1df2f1cc5ba480b12](https://user-images.githubusercontent.com/91971937/193569879-6f5e77a0-7918-4e18-8132-6a6fa903afe4.jpg)

Kind regards, 
Thorin

**Comment by Navid200** [comment]
Those instructions seem to be specific to one brand.

There are manufacturer-specific settings that one needs to address.

There are brands, like Samsung, that are so popular that the developers have got their hands on them and we have them on the guides.

But, it's not practical to include all manufacturer suggestions in our guide.

I'm not sure exactly what you are suggesting.
Do you mind clarifying your suggestion?

Thanks

**Reply by Miquest** [reply]
I have a new smartphone, an Oppo Find X5 Pro, because I read that Oppo has a less aggresive battery saving mode than my OnePlus before. But I found that xDrip does not collect values every 5min with the G6 despite recommended settings. Since these instructions were on dontkillmyapp.com, I thought maybe they could be implemented in an "Oppo Specific" option, or directly in the configuration file.

Regard, 
Thorin

**Reply by Miquest** [reply]
My Problem is the Battery Optimization, I think...

**Reply by Miquest** [reply]
But Oppo should have a less aggressive Optimization, if you trust dontkillmyapp.com

@jamorham , 
What do you think?
